phpunit.xml.dist

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<phpunit bootstrap="./vendor/autoload.php"
+         colors="true"
+         convertErrorsToExceptions="true"
+         convertNoticesToExceptions="true"
+         convertWarningsToExceptions="true"
+         syntaxCheck="true">
+    <testsuites>
+        <testsuite name="unit">
+            <directory suffix="Test.php">./Tests</directory>
```

Comment:
Please follow the boilerplate repo recommendations (just for consistency):

- `tests/` directory, not `Tests`
- Better testsuite name
- Add tests to autoload-dev in composer, it should be a little bit faster that way